src/components/MyAccount/index.tsx

```diff
@@ -246,25 +218,46 @@ export default function MyAccount({ user, organizations, edit, reloadUser, reloa
           />
         </>
       )}
-      <TfMain>
-        <Grid container spacing={3} className={edit ? classes.editContainer : ''}>
-          <Grid item xs={12} className={classes.titleContainer}>
-            <Grid item xs={7}>
-              <h1 className={classes.title}>{strings.MY_ACCOUNT}</h1>
-              <p>{strings.MY_ACCOUNT_DESC}</p>
-            </Grid>
-            <Grid item xs={5} className={classes.centered}>
-              <Button
-                id='edit-account'
-                label={isMobile ? strings.EDIT : dictionary.EDIT_ACCOUNT}
-                onClick={() => history.push(APP_PATHS.MY_ACCOUNT_EDIT)}
-                size='medium'
-                priority='secondary'
-              />
-            </Grid>
-          </Grid>
+      <PageHeaderWrapper nextElement={contentRef.current}>
+        <Box
+          display='flex'
+          justifyContent='space-between'
+          marginBottom={theme.spacing(4)}
+          paddingLeft={theme.spacing(3)}
+        >
+          <Box>
+            <Typography marginBottom={theme.spacing(1.5)} fontSize='24px' fontWeight={600}>
+              {strings.MY_ACCOUNT}
+            </Typography>
+            <Typography fontSize='14px' fontWeight={400}>
+              {strings.MY_ACCOUNT_DESC}
+            </Typography>
```

Comment:
as an aside.. I created a `TitleDescription` component in case it can be reused..
wondering if we should extend that to support these right hand side buttons as well...
we can remove a bunch of boilerplate code across views after that..

Reply:
works great!